js/src/forum/utils/isSafariMobile.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @see https://stackoverflow.com/a/31732310
+ */
+export default function isSafariMobile(): boolean {
+  return (
+    'ontouchstart' in window &&
+    navigator.vendor &&
+    navigator.vendor.indexOf('Apple') > -1 &&
```

Comment:
Can we use `includes` since we don't care about IE anymore?

Reply:
I need to start making it a habit of using includes instead.